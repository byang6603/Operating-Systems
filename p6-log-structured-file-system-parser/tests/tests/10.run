../bin/parse_lfs ls ../sample_files/small-alt.img
