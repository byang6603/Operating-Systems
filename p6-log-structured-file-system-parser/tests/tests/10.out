/
          18 1970-01-01 00:00:00 UTC drw-r--r--    a       
           0 1970-01-01 00:00:00 UTC -rw-r--r--        b       
           0 1970-01-01 00:00:00 UTC -rw-r--r--        c       
           0 1970-01-01 00:00:00 UTC -rw-r--r--        d       
